doc/changelog/11-standard-library/17013-natiter.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  lemmas :g:`iter_swap_gen`, :g:`iter_swap`, :g:`iter_succ`, :g:`iter_succ_r,
+  :g:`iter_add`, :g:`iter_ind`, :g:`iter_rect`, :g:`iter_invariant` for `Nat.iter`.
```

Comment:
Backquote is missing after `iter_succ_r`. Also you should remove the point after `Nat.iter`.

Reply:
Done.